src/main/resources/db/changelog/product-changelog.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="3" author="Wojtek B">
+        <createTable tableName="product">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="price" type="DECIMAL(19, 2)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="quantity" type="INT">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+
+    <changeSet id="8" author="Wojtek B">
```

Comment:
To jest przykład operacji manipulowania danymi (DML), a powyżej masz tworzenie tabelki, czyli przykład DDL. Dobrą praktyką jest rozbijanie tych dwóch rodzajów od siebie, czyli w naszym wypadku:
 - mielibyśmy ogólnie 1 changeset od tworzenia tabelek i constraintów
 - milibyśmy drugi changeset, który zawierałby wszystkie inserty z danymi początkowymi